docs/configuring/README.md

```diff
@@ -1,474 +0,0 @@
-# Configuring ESLint
```

Comment:
Yes, this one was border-line in my mind. While it does rely on code version, it would be updated very rarely (hopefully). I'll add it back.